src/components/ui/mainContent.tsx

```diff
@@ -26,9 +28,36 @@ export default function MainContent({
 
   if (showConnectWalletMsg)
     return (
-      <h1 className="block text-gray-700 font-bold mb-2 text-xl text-center">
-        Please Connect Wallet to proceed.
-      </h1>
+      <div className="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4 flex flex-col items-center">
+        <h1 className="block text-gray-700 font-bold mb-2 text-xl text-center">
+          Welcome to Fil+ Client KYC Portal.
+        </h1>
+        <br />
+        <p className="text-gray-700 text-base text-center">
+          As part of your DataCap Request Application, you are required to
+          complete KYC verification.
+        </p>
+        <p>To do so, please follow these steps:</p>
```

Comment:
```suggestion
        <p>To do so, you'll have to follow these steps:</p>
```

Reply:
Otherwise it suggests user needs to do this before connecting the wallet. And we want them to connect the wallet, as we give links and show current score there.